Src/library/memory_analyzer/widgets/ma_window.e

```diff
@@ -176,12 +181,26 @@ feature {NONE} -- Implementation for agents
 			a_splitter_resize_actions_empty: a_splitter.is_displayed implies a_splitter.resize_actions.is_empty
 		end
 
+
+	update_splitter_proportion_once_dpi_change (a_splitter: EV_SPLIT_AREA; a_proportion: REAL; a_dpi,x, y, w, h: INTEGER_32)
+			-- Set `a_splitter' with `a_proportion'.
+			-- Wipe out `a_splitter' `resize_actions' when `a_splitter' is shown.
+		do
+			update_splitter_proportion_once (a_splitter, a_proportion, x, y, w, h)
+		end
+
 	eiffel_view_frame_size_change (a_x, a_y, a_width, a_height: INTEGER)
 			-- Handle frame change.
 		do
 
 		end
 
+	eiffel_view_frame_dpi_size_change (a_dpi, a_x, a_y, a_width, a_height: INTEGER)
+			-- Handle frame change.
+		do
+
```

Comment:
even if it does nothing , I would call `eiffel_view_frame_size_change` 
this way, if it is redefined, or changed in the future ...

On the other hand, if the `eiffel_view_frame_size_change ` is empty .. could we remove it ?

Reply:
I will use the first option but I will add a TODO to check if it's really used.